website/docs/rules/flutter/avoid-border-all.md

```diff
@@ -0,0 +1,70 @@
+# Avoid using Border.all constructor
+
+![Has auto-fix](https://img.shields.io/badge/-has%20auto--fix-success)
+
+## Rule id {#rule-id}
+
+avoid-border-all
+
+## Severity {#severity}
+
+Performance
```

Comment:
Could you elaborate? It's not obvious at all how this affects performance. If it doesn't, then I'm not even sure, why the users should enable it.

Reply:
This rule suggests replacing widget creation with a const constructor. This will optimize the work of the widget tree build method

Reply:
Ah, I see, didn't even notice `const` usage 👍